apps/meteor/app/api/server/v1/users.ts

```diff
@@ -1023,8 +1024,10 @@ API.v1.addRoute(
 
 			const token = me.services?.resume?.loginTokens?.find((token) => token.hashedToken === hashedToken);
 
+			const loginExp = settings.get<number>('Accounts_LoginExpiration');
+
 			const tokenExpires =
-				(token && 'when' in token && new Date(token.when.getTime() + settings.get<number>('Accounts_LoginExpiration') * 1000)) || undefined;
+				(token && 'when' in token && new Date(token.when.getTime() + getLoginExpiration(loginExp) * 60 * 60 * 24 * 1000)) || undefined;
```

Comment:
Maybe create a helper function (inside the same package) to do this conversion? WDYT?